Cython/Includes/cpython/time.pxd

```diff
@@ -0,0 +1,35 @@
+"""
+Cython implementation of (parts of) the standard library time module.
+"""
+
+cdef extern from "pytime.h":
+    ctypedef _PyTime_t  # alias for int64_t
+    _PyTime_t _PyTime_GetSystemClock() nogil
+    double _PyTime_AsSecondsDouble(_PyTime_t t) nogil
+
+from libc.time cimport (
+    tm,
+    time_t,
+    localtime as libc_localtime,
+)
+
+
+cdef inline double time() nogil:
+    cdef:
+        _PyTime_t tic
+
+    tic = _PyTime_GetSystemClock()
+    return _PyTime_AsSecondsDouble(tic)
+
+
+cdef tm localtime() nogil:
```

Comment:
`.pxd` files can only have `inline` functions since they are not modules by themselves. Modules that cimport these functions must `inline` them on their side.
```suggestion
cdef inline tm localtime() nogil:
```